perllib/FixMyStreet/Cobrand/Gloucestershire.pm

```diff
@@ -51,11 +51,32 @@ sub council_url { 'gloucestershire' }
 
 sub admin_user_domain { 'gloucestershire.gov.uk' }
 
-=item * Allows anonymous reporting
+=item * Allows anonymous reporting (but not for Gloucester City categories)
 
 =cut
 
-sub allow_anonymous_reports { 'button' }
+sub allow_anonymous_reports {
```

Comment:
This function is the same as Default's now, apart from it doesn't have the anonymous_allowed check. So perhaps here instead we could remove this function entirely and tick every category for Gloucestershire to have the same effect without special code?